web/public/locales/en/common.json

```diff
@@ -508,32 +508,32 @@
-    "networksEthicsDescription": "",
+    "helpbuttonsHighlight": "Helpbuttons",
+    "helpbuttonsDescription": "is the open-source software that powers this app. A tool to support communities of any type and local groups in building their digital goods, coordinate them and be tecnologically autonomous. \n\n With Helpbuttons, you can create your own networks for sharing food, housing, transportation, or anything else you have in mind, and then federate with other networks to share users and resources. Helpbuttons specializes in connecting people who don't yet know each other, for various purposes, in the real world.",
+    "forDescription": "We aim to foster a cohesive society, promote cooperation, and optimize the design of collaborative tools in order to fully leverage the advantages of the internet. \n\nIn brief, we've observed the substantial amounts of money, time, and user participation required each time a new collaborative app is launched. This seems unnecessary. Most of these tools share a significant portion of their core functionalities. We believe these commonalities could be managed using the same open software, simply adapted for each unique purpose through various modules.",
+    "helpbuttonDescription": "A Helpbutton refers to a post. Every post published by a user in Helpbuttons is termed a helpbutton. Helpbuttons can represent offers, needs, exchanges, transport proposals, businesses, and so forth. They can also be renamed to suit the specific needs of your app.",
+    "communityDescription": "The Helpbuttons community is the backbone of this project, providing it with sustainability. This tool is designed for adaptability to a multitude of purposes, thereby benefitting a diverse range of users while also encouraging support for the project.\n\nBy federating with other Helpbuttons apps, you can share users and resources.This method of collaboration enables us to unite more people and ultimately improve the tool we provide.\n\nYou can find the source code of this app at <a href=\"https://helpbuttons.org\">Helpbuttons.org</a>",
+    "ethicsDescription": "At Helpbuttons, we're committed to maintaining the highest standards of ethical conduct. We believe in transparency, honesty, and respect for all individuals. \n\nOur software is designed to connect people and facilitate sharing, which is a responsibility we take seriously. We encourage our users to use our platform with the same values in mind, prioritizing respectful communication and mindful sharing. Any misuse of the software, including exploitation, harm to others, or manipulation of the platform's functionalities for personal gain, will not be tolerated. Users are required to adhere to these ethics policies at all times.\n\nIMPORTANT: Please take into account this is an open software and we cannot be responsible of the misuse of this software outside of our servers. If you find a misuse of this software, please contact us immediately.",
```

Comment:
where do you get this texts from? How do you think we can control the misuse of this software? I dont think that it is relevant to have this in the F.A.Q. should be phrased differently.

Reply:
I was redacting according to what I believe we can afford. We can only be responsible for the public use in our servers--- Then we can make another page. But for now it's the place probably. It's easy for the user to have all that info together for now.

Reply:
also i got some of the lines from other apps guidelines